lua/entities/gmod_wire_expression2/core/entity.lua

```diff
@@ -197,14 +197,14 @@ end
 
 --- Returns a vector describing rotation axis, magnitude and sense given as the vector's direction, magnitude and orientation.
 e2function vector entity:angVelVector()
-	if not validPhysics(this) then return self:throw("Invalid entity!", nil) end
+	if not validPhysics(this) then return self:throw("Invalid entity!", {0, 0, 0}) end
 	local phys = this:GetPhysicsObject()
 	return phys:GetAngleVelocity()
 end
 
 --- Specific to env_sun because Source is dum. Use this to trace towards the sun or something.
 e2function vector sunDirection()
-	if not IsValid(sun) then return self:throw("Invalid entity!", nil) end
+	if not IsValid(sun) then return {0, 0, 0} end
```

Comment:
Shouldn't there still be a throw here?

Reply:
It wouldn't really make sense as it's not the users fault that lua couldn't find the sun if it isn't valid here.